mod/caching/set/type_plus_right/wikirate_company/wikirate_topic.rb

```diff
@@ -17,7 +17,7 @@ def company_name
   Answer.select(:company_id).where(metric_id: metric_id).uniq
         .pluck(:company_id).map do |company_id|
     # faster way to get this from company+topic?
-    Card.fetch(company_id.cardname.trait :wikirate_topic)
+    Card.fetch(company_id.cardname.trait(:wikirate_topic))
```

Comment:
do we have a style preference for these situations?

Reply:
inner parenthesis are mandatory, outer ones are optional